crates/next-custom-transforms/src/transforms/server_actions.rs

```diff
@@ -2891,6 +2935,14 @@ fn emit_error(error_kind: ServerActionsErrorKind) {
                 "#
             },
         ),
+        ServerActionsErrorKind::UnknownCacheKind { span, cache_kind } => (
+            span,
+            formatdoc! {
+                r#"
+                    Unknown cache kind "{cache_kind}". Please configure a cache handler for this kind in the experimental "cacheHandlers" object in your Next.js config.
```

Comment:
i think we usually refer to these options as `experimental.someOption` in diagnostics

```suggestion
                    Unknown cache kind "{cache_kind}". Please configure a cache handler for this kind in the `experimental.cacheHandlers` object in your Next.js config.
```